echopype/convert/set_groups_ek80.py

```diff
@@ -236,17 +236,27 @@ def set_sonar(self, beam_group_type: list = ["power", None]) -> xr.Dataset:
                     "standard_name": "sound_frequency",
                 },
             ),
-            "serial_number": (["channel"], var["serial_number"]),
-            "transducer_name": (["channel"], var["transducer_name"]),
-            "sonar_serial_number": (["channel"], var["channel_id_short"]),
-            "sonar_software_name": (
+            "sonar_serial_number": (
                 ["channel"],
-                var["application_name"],
-            ),  # identical for all channels
-            "sonar_software_version": (
+                var["channel_id_short"],
+                {
+                    "long_name": "Sonar serial number",
+                },
+            ),
+            "transducer_serial_number": (
+                ["channel"],
+                var["serial_number"],
+                {
+                    "long_name": "Transducer serial number",
+                },
+            ),
+            "transducer_name": (
                 ["channel"],
-                var["application_version"],
-            ),  # identical for all channels
+                var["transducer_name"],
+                {
+                    "long_name": "Transducer name",
+                },
+            ),
```

Comment:
I think something is not quite right with these changes. 

- `channel_id_short` are the transducer model plus serial number
- `transducer_name` contains the transducer model. 
- `serial_number` contains the transceiver serial number (transceivers are things like GPT and WBT).

See https://github.com/OSOceanAcoustics/echopype/issues/212#issuecomment-1053626927 for example printouts, I think you could see how these numbers are combined to become what's in `channel_id`

Reply:
To clarify, my changes in the block you're commenting on amount to just two things:
- Rename `serial_number` to `transducer_serial_number`
- Add a `long_name` attribute with values that are just the variable name spelled out.

I'm not sure what changes you're proposing. But since `transducer_name` and `serial_number` are variable names from echopype, not the convention, we can rename them as we see fit. Are you proposing we rename them to `transducer_model` and `transceiver_serial_number`, respectively?

As for `sonar_serial_number`, that's defined as a global attribute in the convention and as a variable by echopype, for EK80 only b/c it varies with channel. In your examples in #212, it's "0" in one case (which I interpret as not available / known) and "202" in another (a suspiciously short value for a serial number, IMHO). Should we use that parameter instead of `channel_id_short`? In https://github.com/OSOceanAcoustics/echopype/issues/212#issuecomment-1099298069 you had equated transducer serial number with `channel_id_short`.

Reply:
My comment above is to clarify what each of the variables are coming out from the parser, and that transceiver and transducer are different things.

I think `sonar_serial_number` is ill-defined since it can mean too many things. 

Below is what I propose:
- store the parsed `transducer_name` as `transducer_name` in echopype (ie. no change), so that it is easy for folks familiar with EK80 raw data know what they are
- store the parsed `serial_number` as `transceiver_serial_number` in echopype, so that it is specific that it is not the transducer serial number
- store the parsed `transducer_serial_number` as `transducer_serial_number` in echopype (ie. no change), for clarify
- do not store `sonar_serial_number` for EK80: This is marked as recommended and not mandatory in the convention. I think it is confusing what it is in this context. And, not populating it makes it consistent with EK60. 😬

`channel_id_short` seems a combination of `transducer_name` and `transducer_serial_number`, so I am thinking that we can safely ignore it.

Reply:
I've pushed a commit that implements this:
- `sonar_serial_number` is an empty global attribute, no longer a variable
- variables:
  - `transducer_name`, based on parser `transducer_name` parameter
  - `transducer_serial_number`, based on parser `transducer_serial_number` parameter
  - `transceiver_serial_number`, based on parser `serial_number` parameter

That follows what you recommended, but please review.